### PR TITLE
Dashboard editor accessibility fixes. Several Dutch 🇳🇱  labels

### LINKED ISSUE #1992: Dashboard Accessibility contrast issues
image

### PR DESCRIPTION
**Pull request by I-Valchev**
Fixes #1992

### CODE REVIEW

File: translations/messages.nl.xlf [hunk 1]
```diff
@@ -1776,5 +1776,23 @@
         <target>Weet je zeker dat je deze content wilt verwijderen?</target>
       </segment>
     </unit>
+    <unit id="RMPnq6o" name="label.username_or_email">
+      <segment>
+        <source>label.username_or_email</source>
+        <target>Gebruikersnaam of email</target>
```
Comment (bobdenotter) [inline translations/messages.nl.xlf:1782]: Dutch is stupid.

```suggestion
        <target>Gebruikersnaam of e-mail</target>
```

File: translations/messages.nl.xlf [hunk 2]
```diff
@@ -1776,5 +1776,23 @@
...
+      </segment>
+    </unit>
+    <unit id="z_tOHwq" name="placeholder.username_or_email">
+      <segment>
+        <source>placeholder.username_or_email</source>
+        <target>uw gebruikersnaam of e-mailadres</target>
```
Comment (bobdenotter) [inline translations/messages.nl.xlf:1788]: We use informal pronouns. 

```suggestion
        <target>je gebruikersnaam of e-mailadres</target>
```

File: translations/messages.nl.xlf [hunk 3]
```diff
@@ -1776,5 +1776,23 @@
...
+      </segment>
+    </unit>
+    <unit id="xgYhRkc" name="placeholder.password">
+      <segment>
+        <source>placeholder.password</source>
+        <target>uw wachtwoord</target>
```
Comment (bobdenotter) [inline translations/messages.nl.xlf:1794]: Dito

```suggestion
        <target>je wachtwoord</target>
```

**Review by bobdenotter — CHANGES_REQUESTED** [review]

File: translations/messages.nl.xlf (hunk 1 above)
Comment (I-Valchev) [reply translations/messages.nl.xlf:1782]: Argh.. that reminds me of people in Bulgaria who refer to it not as "email" but "electronic post" :sweat_smile:   *cringe*

**Review by bobdenotter — APPROVED** [review]

**Pull request merged**